src/sfizz/Region.cpp

```diff
@@ -59,6 +59,12 @@ bool sfz::Region::parseOpcode(const Opcode& opcode)
     case hash("offset_random"):
         setValueFromOpcode(opcode, offsetRandom, Default::offsetRange);
         break;
+    case hash("offset_cc&"):
+        if (opcode.parameters.back() > config::numCCs)
```

Comment:
I note there exist multiple occurrences of `ccNumber > config::numCCs` in code.
There is also `for (int cc = 0; cc < config::numCCs; cc++)`.
One of these has to be wrong. Perhaps instead the checks should use `>=`.

Reply:
You're right it should be `>=` in all these checks.